contents/handbook/people/team-structure/app-small-teams.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Product Small Teams
```

Comment:
feedback: this title confused me! (ie we treat website and docs like a product too / customer success also should be treated like a product)

I'd probably just work this content into the existing page that explains small teams: https://posthog.com/handbook/people/team-structure/team-structure

I generally think though the content here is valuable and clarifies a bunch of things!

Reply:
+1 to that. I'd say we're "small teams", and making a distinction "product small teams", "infra small teams", "marketing small teams", etc loses the "everyone's a full stack product minded engineer" mindset. Also, infra, marketing, customer success, etc are all part of our product.

Reply:
Good suggestions and I agree with the reasoning. I'll refactor into the team-structure page